std_msgs/Header header
bool[] emergency
int32 my_pos_x
int32 my_pos_y
int32 my_degree
int32 enemy1_x
int32 enemy1_y 
int32 enemy2_x
int32 enemy2_y
int32 ally_x
int32 ally_y
int32 servo_state
int32 stepper
int32 hand
int32 status
int32 strategy
float32 time 
int32 ns 
int32[] cup_color

